index.html

```diff
@@ -13,6 +13,7 @@
       <div class="container-book">
         <div class="container-img">
           <div class="cover-image">
+            <img class="cover-image img" src="./assets/book-cover.png" alt="img">
```

Comment:
In the file index.html the class `cover-image` is assigned to both the parent div and the image.
After a little peek in the .scss I think `cover-image` is unnecessary for the image element

Reply:
Related to [comment](https://github.com/wolox-training/ng-angularjs/pull/2#discussion_r591769201), The ".img" class is assigned to the img element but in the .scss only the img html tag is being targeted.